openassessment/xblock/static/js/spec/lms/oa_staff_area.js

```diff
@@ -155,11 +157,12 @@ describe('OpenAssessment.StaffAreaView', function() {
 
     var verifyAssessType = function(staffArea, assessType) {
         expect(staffArea.callStaffAssess).toHaveBeenCalledWith(
-                jasmine.any(String), jasmine.any(Object), jasmine.any(Object), jasmine.any(Function), jasmine.any(String), assessType
+            jasmine.any(String), jasmine.any(Object), jasmine.any(Object),
+            jasmine.any(Function), jasmine.any(String), assessType
         );
     };
 
-    var verifyFocused = function (element) {
```

Comment:
Just an observation: was it manual restyling to remove the spaces or some automated tool?

Reply:
I manually restyled wherever `eslint` was complaining in this file. All restyling is contained within [this commit](https://github.com/edx/edx-ora2/pull/1399/commits/82e07c80f9e163eb899172592925df9a387e6a31).

Reply:
Some styling it enforces:
- Specific spacing between keywords and brackets
- `'` instead of `"` for strings
- No unused vars
- No use of `this` in unbound functions
- Max line length
- Trailing commas for object key/value pairs
- No use of capital variable names (except constructors)